packs/feats/tempest-gaze.json

```
@@ -28,7 +28,36 @@
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "mergeable": true,
+                "option": "storming-gaze",
+                "suboptions": [
+                    {
+                        "label": "{item|name}",
+                        "value": "tempest-gaze"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "itemType": "feat",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "predicate": [
+                    "item:slug:storming-gaze",
+                    "storming-gaze:tempest-gaze"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Oni.StormingGaze.TempestGaze"
```

```suggestion
                        "text": "PF2E.SpecificRule.Oni.StormingGaze.TempestGaze"
```
We can probably just use the description rather than localizing it. `{item|system.description.value}`

---

That's fair, in that case it could add to Storming Gaze instead of overriding it